### ISSUE TITLE
Alchemical Reaction Chamber Texture loss when i put Sanguine Reverter in

### ISSUE THREAD

**Issue by MindOfNasser** [AllTheMods/ATM-9#923]
**Hey, just want to give heads up about it it does not effect the progress or something just looks stupid lol, tried with both w/shader without shader same issue**

w/reverter
![Screenshot_1](https://github.com/AllTheMods/ATM-9/assets/45881640/a74b41c5-5434-4db0-b9b3-54774bb3d518)

without the reverter
![Screenshot_2](https://github.com/AllTheMods/ATM-9/assets/45881640/a5379260-51a4-4170-b0e2-826dcda90e5b)

**Comment by MindOfNasser** [comment]
Blood Magic mod btw, sorry forgot to mention the mod name..

**Comment by HyperFirez** [comment]
Ditto. You didn't add any extra mods to this pack did you? I did, so I want to see if one of the mods I added caused it or not.

**Comment by TheBedrockMaster** [comment]
Yeah, I believe this is a Blood Magic bug.

**Comment by unlimitedsola** [comment]
This issue still exists in the latest version of the modpack as of now (1.1.1).

Tested with just the BloodMagic mod alone (same version as in the modpack: `3.3.3-45`) and it renders correctly, thus it is likely caused by interactions from other mods.

<img width="932" height="828" alt="Image" src="https://github.com/user-attachments/assets/2e241c66-98ff-49c7-96ea-1bdeea5ff878" />

**Comment by TheBedrockMaster** [comment]
Yes, however I don't believe this is something we can fix on our end.
